Model/Cron.php

```diff
@@ -1181,7 +1189,8 @@ protected function _processNotification()
                 $isOrderCc = strcmp(
                         $this->_paymentMethodCode(),
                         'adyen_cc'
-                    ) == 0 || strcmp($this->_paymentMethodCode(), 'adyen_oneclick') == 0;
+                    ) == 0 || strcmp($this->_paymentMethodCode(), 'adyen_oneclick') == 0
+                    || $this->paymentMethodsHelper->isWalletPaymentMethod($notificationPaymentMethod);
```

Comment:
Won't it be better to add the wallet as a separate check? Like `$isWallet`. When you add this to the `$isOrderCc` variable it reads as if wallet payments are also CC payments.